apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -106,9 +106,9 @@ internal async Task<SpannerDataReader> ExecuteReaderAsync(CommandBehavior behavi
                 ValidateConnectionAndCommandTextBuilder();
                 ValidateCommandBehavior(behavior);
 
-                if (CommandTextBuilder.SpannerCommandType != SpannerCommandType.Select && CommandTextBuilder.SpannerCommandType != SpannerCommandType.Read)
+                if (CommandTextBuilder.SpannerCommandType != SpannerCommandType.Select && CommandTextBuilder.SpannerCommandType != SpannerCommandType.Read && CommandTextBuilder.SpannerCommandType != SpannerCommandType.Dml)
                 {
-                    throw new InvalidOperationException("ExecuteReader functionality is only available for queries and reads.");
+                    throw new InvalidOperationException("ExecuteReader functionality is only available for queries, reads and dmls.");
```

Comment:
DML being an acronym, I will probably use DMLs here as it will surface to the end user.